### PR TITLE
feat: app launchpad i18n

### PR DESCRIPTION
**Pull request by zjy365**
### Summary
🌐🗣️🛠️

This pull request adds internationalization and localization support to the frontend application of the applaunchpad provider, using the `i18next` framework and its integrations with `React` and the browser. It adds two new files: `public/locales/en/translation.json` and `public/locales/zh/translation.json`, which contain the English and Chinese translations of the strings used in the frontend. It also updates several components and hooks to use the `useTranslation` hook and the `t` function from `react-i18next` to render translated strings based on the current language. Additionally, it adds a new component: `NoSSR`, which prevents server-side rendering of components that depend on browser-specific features.

> _We're breaking the language barrier, we're `i18next`_
> _We're using the `t` function, we're `react-i18next`_
> _We're translating every string, we're `useTranslation`_
> _We're internationalizing the frontend, we're `translation.json`_

### Walkthrough
*  Add and configure i18next and related libraries for internationalization and localization of the frontend application ([link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-a840060e71933fd186996b2fc7b99d4a6736fc6a3270aefe7a12c39274a5e2fcR27-R29),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-a840060e71933fd186996b2fc7b99d4a6736fc6a3270aefe7a12c39274a5e2fcR41),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-a88c6ea140f1b6c03ad2f356e6a4e18a8990c22a6c2bd948c241bee715433badL1-R122),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-1eb95aad1f46b9a926d0fd3ca9ec0f866ae91257200837c26b527f1ed82f46e6L1-R122),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-ebd69d41e82f5d5fbc4c8884ac0789e5e3e743e88c10031cebbf19d53120ea02R16))
*  Use the useTranslation hook and the t function to render translated strings for the keys that match the values of the labels, titles, messages, and texts in the JSX of various components ([link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-9994c9a6f430dad4c6b71d06ce9262b2bd1721023cb7bf627b36628c45549132R5),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-9994c9a6f430dad4c6b71d06ce9262b2bd1721023cb7bf627b36628c45549132R16),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-9994c9a6f430dad4c6b71d06ce9262b2bd1721023cb7bf627b36628c45549132L32-R34),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-3e7b9ef98937541e1eef60baa6fb8d8a48e11d0fc2d7c6c590254d722d017690R1-R8),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-b86d180b364a4e9eb21524b1553bb83f359d4555e80ae8820c3c231444f8830dL5-R5),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-b86d180b364a4e9eb21524b1553bb83f359d4555e80ae8820c3c231444f8830dR18),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-b86d180b364a4e9eb21524b1553bb83f359d4555e80ae8820c3c231444f8830dL73-R74),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-a8d1cd8f74277e529d1bbc1f6a803178d433d3e558cab9fe5c1184ca7e7aa0d5R3),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-a8d1cd8f74277e529d1bbc1f6a803178d433d3e558cab9fe5c1184ca7e7aa0d5R16),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-a8d1cd8f74277e529d1bbc1f6a803178d433d3e558cab9fe5c1184ca7e7aa0d5L34-R36),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-c492540c760f8684a6918536224aa581df7fdd90a80270c7ad5120ef3b845b7dR4),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-c492540c760f8684a6918536224aa581df7fdd90a80270c7ad5120ef3b845b7dR15),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-c492540c760f8684a6918536224aa581df7fdd90a80270c7ad5120ef3b845b7dL66-R68),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-1ddd82d58fa77f7c18f8ca4bdbf2be32f32c13f306cbf403f37de94f9fc2a686R3),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-1ddd82d58fa77f7c18f8ca4bdbf2be32f32c13f306cbf403f37de94f9fc2a686R13),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-1ddd82d58fa77f7c18f8ca4bdbf2be32f32c13f306cbf403f37de94f9fc2a686L65-R67),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-4a09710777f87a4a9d392168b7f6f30554a488572a017e2a3d59297c76f1808eL22-R25),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-4a09710777f87a4a9d392168b7f6f30554a488572a017e2a3d59297c76f1808eL44-R53),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-4a09710777f87a4a9d392168b7f6f30554a488572a017e2a3d59297c76f1808eL57-R72),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-4a09710777f87a4a9d392168b7f6f30554a488572a017e2a3d59297c76f1808eL72-R82),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-4a09710777f87a4a9d392168b7f6f30554a488572a017e2a3d59297c76f1808eL84-R92),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-4a09710777f87a4a9d392168b7f6f30554a488572a017e2a3d59297c76f1808eL98-R106),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-4a09710777f87a4a9d392168b7f6f30554a488572a017e2a3d59297c76f1808eL112-R120),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-4a09710777f87a4a9d392168b7f6f30554a488572a017e2a3d59297c76f1808eL124-R132),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-4a09710777f87a4a9d392168b7f6f30554a488572a017e2a3d59297c76f1808eL151-R164),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-4a09710777f87a4a9d392168b7f6f30554a488572a017e2a3d59297c76f1808eL166-R174),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-4a09710777f87a4a9d392168b7f6f30554a488572a017e2a3d59297c76f1808eL179-R187),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-4a09710777f87a4a9d392168b7f6f30554a488572a017e2a3d59297c76f1808eL193-R201),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-4a09710777f87a4a9d392168b7f6f30554a488572a017e2a3d59297c76f1808eL232-R240),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-4a09710777f87a4a9d392168b7f6f30554a488572a017e2a3d59297c76f1808eL283-R291),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-ad210638f261dbbc4ba1deb17e22e093e7b6bb7578c3f0b58f46b43d8c5152a8L11-R14),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-ad210638f261dbbc4ba1deb17e22e093e7b6bb7578c3f0b58f46b43d8c5152a8L40-R46),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-ad210638f261dbbc4ba1deb17e22e093e7b6bb7578c3f0b58f46b43d8c5152a8L65-R68),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-ad210638f261dbbc4ba1deb17e22e093e7b6bb7578c3f0b58f46b43d8c5152a8L75-R78),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-ad210638f261dbbc4ba1deb17e22e093e7b6bb7578c3f0b58f46b43d8c5152a8L89-R92),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-ad210638f261dbbc4ba1deb17e22e093e7b6bb7578c3f0b58f46b43d8c5152a8L115-R118),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-ad210638f261dbbc4ba1deb17e22e093e7b6bb7578c3f0b58f46b43d8c5152a8L141-R144),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-a33e61011a16b6c2eeedceb9fc1c09550cf9f66d7f7c416f835336f86ca3956cR17),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-a33e61011a16b6c2eeedceb9fc1c09550cf9f66d7f7c416f835336f86ca3956cR28),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-a33e61011a16b6c2eeedceb9fc1c09550cf9f66d7f7c416f835336f86ca3956cL56-R70),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-a33e61011a16b6c2eeedceb9fc1c09550cf9f66d7f7c416f835336f86ca3956cL79-R83),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-a33e61011a16b6c2eeedceb9fc1c09550cf9f66d7f7c416f835336f86ca3956cL89-R93),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-134f94c26d993fe5ecf63494e11b4cd294dfbc24ae94a78aff78e154af844df4R13),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-134f94c26d993fe5ecf63494e11b4cd294dfbc24ae94a78aff78e154af844df4R32),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-134f94c26d993fe5ecf63494e11b4cd294dfbc24ae94a78aff78e154af844df4L123-R126),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-134f94c26d993fe5ecf63494e11b4cd294dfbc24ae94a78aff78e154af844df4L141-R144),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-134f94c26d993fe5ecf63494e11b4cd294dfbc24ae94a78aff78e154af844df4L154-R157),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-134f94c26d993fe5ecf63494e11b4cd294dfbc24ae94a78aff78e154af844df4L170-R173),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-134f94c26d993fe5ecf63494e11b4cd294dfbc24ae94a78aff78e154af844df4L185-R188),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-134f94c26d993fe5ecf63494e11b4cd294dfbc24ae94a78aff78e154af844df4L200-R203),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-5a27e7c437ed17392fdad93812b050ea973011305f00c21703a171e91620bbd0R21),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-5a27e7c437ed17392fdad93812b050ea973011305f00c21703a171e91620bbd0R38),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-5a27e7c437ed17392fdad93812b050ea973011305f00c21703a171e91620bbd0L112-R114),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-5a27e7c437ed17392fdad93812b050ea973011305f00c21703a171e91620bbd0L140-R142),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-1cc4abd9b73af6bc796186d3c7da03fe818946f202898867a92c313ddda3511dR31),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-1cc4abd9b73af6bc796186d3c7da03fe818946f202898867a92c313ddda3511dR46),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-1cc4abd9b73af6bc796186d3c7da03fe818946f202898867a92c313ddda3511dL158-R160),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-1cc4abd9b73af6bc796186d3c7da03fe818946f202898867a92c313ddda3511dL197-R200),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-1cc4abd9b73af6bc796186d3c7da03fe818946f202898867a92c313ddda3511dL208-R211),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-1cc4abd9b73af6bc796186d3c7da03fe818946f202898867a92c313ddda3511dL218-R221),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-1cc4abd9b73af6bc796186d3c7da03fe818946f202898867a92c313ddda3511dL258-R261),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-1cc4abd9b73af6bc796186d3c7da03fe818946f202898867a92c313ddda3511dL306-R309),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-1cc4abd9b73af6bc796186d3c7da03fe818946f202898867a92c313ddda3511dL319-R322))
*  Change the label values of the appStatusMap object from Chinese strings to English strings, so they can be used as keys for the translation function ([link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-0a605d37fdeda123d6b79fc4f83d02530455a26943b8a74c759751dae4ac1f13L9-R9),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-0a605d37fdeda123d6b79fc4f83d02530455a26943b8a74c759751dae4ac1f13L16-R16),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-0a605d37fdeda123d6b79fc4f83d02530455a26943b8a74c759751dae4ac1f13L23-R23),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-0a605d37fdeda123d6b79fc4f83d02530455a26943b8a74c759751dae4ac1f13L30-R30))
*  Change the return values of the editModeMap function from Chinese strings to English strings, so they can be used as keys for the translation function ([link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-8bea0ee10fef94a8ce2bdd1ecf8b746dc58de576d14ecad23c687fa63e46817bL6-R19))
*  Change the parameters of the useConfirm hooks from Chinese strings to English strings, so they can be used as keys for the translation function ([link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-a48dcb3bbd764ece830a17123b84d94692095b980bdd3eae09eaa552cc3362d9L12-R15),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-a48dcb3bbd764ece830a17123b84d94692095b980bdd3eae09eaa552cc3362d9L36-R41),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-a48dcb3bbd764ece830a17123b84d94692095b980bdd3eae09eaa552cc3362d9L49-R51),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-a48dcb3bbd764ece830a17123b84d94692095b980bdd3eae09eaa552cc3362d9L59-R61),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-a48dcb3bbd764ece830a17123b84d94692095b980bdd3eae09eaa552cc3362d9L66-R68),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-ebd69d41e82f5d5fbc4c8884ac0789e5e3e743e88c10031cebbf19d53120ea02L39-R41),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-134f94c26d993fe5ecf63494e11b4cd294dfbc24ae94a78aff78e154af844df4L39-R45))
*  Change the title values of the toast functions from Chinese strings to English strings, so they can be used as keys for the translation function ([link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-a33e61011a16b6c2eeedceb9fc1c09550cf9f66d7f7c416f835336f86ca3956cL37-R39),[link](https://github.com/labring/sealos/pull/3235/files?diff=unified&w=0#diff-134f94c26d993fe5ecf63494e11b4cd294dfbc24ae94a78aff78e154af844df4L52-R55))

### CODE REVIEW

File: frontend/providers/applaunchpad/public/locales/en/translation.json [hunk 1]
```diff
@@ -0,0 +1,123 @@
+{
+  "Applications": "Applications",
+  "New Application": "New Application",
```
Comment (fanux) [inline frontend/providers/applaunchpad/public/locales/en/translation.json:3]: Create Application

File: frontend/providers/applaunchpad/public/locales/en/translation.json [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  "Operation": "Operation",
+  "Running": "Running",
+  "Paused": "Paused",
+  "Active": "Active",
+  "Details": "Details",
+  "Start Up": "Start Up",
+  "Delete": "Delete",
+  "Pause": "Pause",
+  "Restart": "Restart",
+  "Change": "Change",
```
Comment (fanux) [inline frontend/providers/applaunchpad/public/locales/en/translation.json:22]: Update

File: frontend/providers/applaunchpad/public/locales/en/translation.json [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+  "ApplicationType": "ApplicationType",
```
Comment (fanux) [inline frontend/providers/applaunchpad/public/locales/en/translation.json:23]: Application Type

File: frontend/providers/applaunchpad/public/locales/en/translation.json [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+  "Basic Information": "Basic Information",
+  "Deployment Mode": "Deployment Mode",
+  "Advanced Configuration": "Advanced Configuration",
+  "Start Command": "Start Command",
```
Comment (fanux) [inline frontend/providers/applaunchpad/public/locales/en/translation.json:27]: Command

File: frontend/providers/applaunchpad/public/locales/en/translation.json [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+  "Run Parameters": "Run Parameters",
```
Comment (fanux) [inline frontend/providers/applaunchpad/public/locales/en/translation.json:28]: Parameters

File: frontend/providers/applaunchpad/public/locales/en/translation.json [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+  "Not Configured": "Not Configured",
+  "Accessible to the Public": "Accessible to the Public",
+  "Configuration File": "Configuration File",
+  "Storage Volume": "Storage Volume",
+  "Environment Variables": "Environment Variables",
+  "Fixed Number of Instances": "Fixed Number of Instances",
```
Comment (fanux) [inline frontend/providers/applaunchpad/public/locales/en/translation.json:34]: Number of Instances

File: frontend/providers/applaunchpad/public/locales/en/translation.json [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+  "Image Name": "Image Name",
+  "Private": "Private",
+  "Public": "Public",
+  "Image Name (Private)": "Image Name (Private)",
+  "Real-time Monitoring": "Real-time Monitoring",
+  "Network Configuration": "Network Configuration",
+  "Intranet Address": "Intranet Address",
+  "Internet Address": "Internet Address",
```
Comment (fanux) [inline frontend/providers/applaunchpad/public/locales/en/translation.json:42]: duplicated

File: frontend/providers/applaunchpad/public/locales/en/translation.json [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+  "Log": "Log",
+  "None": "None",
+  "Intelligent Analysis": "Intelligent Analysis",
+  "Problem Analysis": "Problem Analysis",
+  "Export": "Export",
+  "Edit Environment Variables": "Edit Environment Variables",
+  "Application Deployment": "Application Deployment",
+  "Creating": "Creating",
+  "Abnormality Detected": "Abnormality Detected",
+  "Change Application": "Change Application",
```
Comment (fanux) [inline frontend/providers/applaunchpad/public/locales/en/translation.json:56]: Update Application

File: frontend/providers/applaunchpad/public/locales/en/translation.json [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+  "Confirm Deploy Application?": "Confirm Deploy Application?",
+  "Deployment Successful": "Deployment Successful",
+  "Deployment Failed": "Deployment Failed",
+  "Update Application": "Update Application",
+  "Confirm Update Application?": "Confirm Update Application?",
+  "Update Successful": "Update Successful",
+  "Update Failed": "Update Failed",
+  "Prompt": "Prompt",
+  "jump_prompt": "Jump prompt",
+  "jump_message": "This application does not allow standalone usage. Click OK to proceed to Sealos Desktop.",
```
Comment (fanux) [inline frontend/providers/applaunchpad/public/locales/en/translation.json:67]: This application cannot be used independently. Please click 'OK' to proceed to Sealos Desktop.

File: frontend/providers/applaunchpad/public/locales/en/translation.json [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+  "Please note that application cannot be modified while in paused state, and if you are using storage volumes, storage charges will still apply. Please confirm!": "Please note that application cannot be modified while in paused state, and if you are using storage volumes, storage charges will still apply. Please confirm!",
```
Comment (fanux) [inline frontend/providers/applaunchpad/public/locales/en/translation.json:68]: The resources occupied by storage will continue to be charged.

File: frontend/providers/applaunchpad/public/locales/en/translation.json [hunk 11]
```diff
@@ -0,0 +1,123 @@
...
+  "Confirm to restart this application?": "Confirm to restart this application?",
+  "You haven't created any application yet.": "You haven't created any application yet.",
+  "Confirm deletion": "Confirm deletion",
+  "Deletion warning": "Deletion warning",
+  "Are you sure you want to delete this application? If you proceed, all data for this project will be deleted.": "Are you sure you want to delete this application? If you proceed, all data for this project will be deleted.",
+  "Please enter": "Please enter",
+  "Elastic scaling": "Elastic scaling",
```
Comment (fanux) [inline frontend/providers/applaunchpad/public/locales/en/translation.json:75]: Auto scaling

File: frontend/providers/applaunchpad/public/locales/en/translation.json [hunk 12]
```diff
@@ -0,0 +1,123 @@
...
+  "Memory target value": "Memory target value",
+  "Reboot Success": "Reboot Success",
+  "success": "success",
+  "target_value": "target_value",
+  "Edit Env Variable": "Edit Env Variable",
+  "public": "public",
+  "private": "private",
+  "Config Form": "Config Form",
+  "file": "file",
+  "Mirror Source": "Mirror Source",
```
Comment (fanux) [inline frontend/providers/applaunchpad/public/locales/en/translation.json:93]: Mirror Source->Image

File: frontend/providers/applaunchpad/public/locales/en/translation.json [hunk 13]
```diff
@@ -0,0 +1,123 @@
...
+  "filename": "filename",
+  "file value": "file value",
+  "local storage": "local storage",
+  "multiple instances do not share data": "multiple instances do not share data",
+  "capacity": "capacity",
+  "mount path": "mount path",
+  "Expose Container Ports": "Expose Container Ports",
+  "CPU target is the CPU utilization rate of any container": "CPU target is the CPU utilization rate of any container",
+  "protocol": "protocol",
+  "exportDomain": "exportDomain",
```
Comment (fanux) [inline frontend/providers/applaunchpad/public/locales/en/translation.json:106]: Export Domain

File: frontend/providers/applaunchpad/public/locales/en/translation.json [hunk 14]
```diff
@@ -0,0 +1,123 @@
...
+  "Custom domain": "Custom domain",
+  "Please CNAME your custom domain to": "Please CNAME your custom domain to",
+  "Option": "Option",
+  "Run command": "Run command",
```
Comment (fanux) [inline frontend/providers/applaunchpad/public/locales/en/translation.json:110]: Command

Comment (fanux) [reply frontend/providers/applaunchpad/public/locales/en/translation.json:110]: OR Exec command

File: frontend/providers/applaunchpad/public/locales/zh/translation.json [hunk 15]
```diff
@@ -0,0 +1,123 @@
...
+  "Config Form": "配置表单",
+  "file": "文件",
+  "Mirror Source": "镜像源",
+  "App Name": "应用名称",
+  "Basic Config": "基础配置",
+  "YAML File": "YAML 文件",
+  "filename": "文件名",
+  "file value": "文件值",
+  "local storage": "本地存储",
+  "multiple instances do not share data": "多个实例间数据不互通",
```
Comment (fanux) [inline frontend/providers/applaunchpad/public/locales/zh/translation.json:100]: 存储卷不支持多个实例共享

File: frontend/providers/applaunchpad/public/locales/zh/translation.json [hunk 16]
```diff
@@ -0,0 +1,123 @@
...
+  "capacity": "容量",
+  "mount path": "挂载路径",
+  "Expose Container Ports": "容器暴露端口",
+  "CPU target is the CPU utilization rate of any container": "CPU 目标值为任一容器的 CPU 利用率",
+  "protocol": "协议",
+  "exportDomain": "出口域名",
+  "Custom domain": "自定义域名",
+  "Please CNAME your custom domain to": "请将您的自定义域名 cname 到",
+  "Option": "选填",
+  "Run command": "运行命令",
```
Comment (fanux) [inline frontend/providers/applaunchpad/public/locales/zh/translation.json:110]: duplicated

File: frontend/providers/applaunchpad/public/locales/en/translation.json (hunk 7 above)
Comment (zjy365) [reply frontend/providers/applaunchpad/public/locales/en/translation.json:42]: One internal network address and one external network address, I changed it to External Address

File: frontend/providers/applaunchpad/public/locales/zh/translation.json (hunk 16 above)
Comment (zjy365) [reply frontend/providers/applaunchpad/public/locales/zh/translation.json:110]: The Chinese commands corresponding to Command are different, one is the start command, and the other is the run command, which should be uniformly changed to Command

File: frontend/desktop/src/components/LangSelect/index.tsx [hunk 17]
```diff
@@ -44,6 +45,7 @@ const LangSelect: React.FC<MenuButtonProps> = (props) => {
             fontSize="sm"
             {...(key === i18n.language ? { bg: 'A7Gray.200' } : {})}
             onClick={() => {
+              setCookie('NEXT_LOCALE', key, { expires: 7 });
```
Comment (zzjin) [inline frontend/desktop/src/components/LangSelect/index.tsx:48]: locale can be set to logner ,like 1y.

**Review by zzjin — APPROVED** [review]

**Pull request merged**